### ISSUE TITLE
Tab G and other features not working 

### ISSUE THREAD

**Issue by rbotts2021** [ShunSakurai/asana-navigator#7]
**To Reproduce**
the extension is installed on both firefox and Chrome
1. Go to any task
3. Tab+G
4. Nothing happens
5. Open hamburger menu in task, Tab +G Make a subtask is not listed  (other functions missing too)

![Screenshot 2022-09-07 030920](https://user-images.githubusercontent.com/77405812/188812441-6154696c-7c14-4b2a-9321-de8f1fe099dc.jpg)

 - OS: Win 10
 - Browser version: 
Google Chrome
Chrome is up to date
Version 105.0.5195.102 (Official Build) (64-bit)

Firefox 
104.0.2

Could something on both browsers be blocking the extension?

**Comment by ShunSakurai** [comment]
Thank you for the bug report! I'm aware of the isuses. I'll work on this weekend 🙇

**Comment by rbotts2021** [comment]
Thank you so much! Asana navigator is my LIFE SAVER!
________________________________

**Comment by rbotts2021** [comment]
@ShunSakurai   A little harder to fix than we hoped huh?

**Comment by ShunSakurai** [comment]
Yes, much harder than I thought. It seems many features including keyboard shortcuts are now blocked by Asana.

**Comment by ShunSakurai** [comment]
Update: I found that defining the keyboard shortcuts under "commands" in manifest.json works.

**Comment by rbotts2021** [comment]
is that something the user needs to update, or that you will update in the code?

**Comment by ShunSakurai** [comment]
I'll make massive changes to the code.
